.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": [["@babel/plugin-proposal-decorators", { "legacy": true }]]
```

Comment:
tutaj sie zastanawiam czy nie trzeba zmienić tego na ->
`{
    "presets": [
        "@babel/preset-react"
    ]
}`
najlepiej jakbys przetestowal jak sie z tym zachowuje

Reply:
Działa bez zarzutów

Reply:
testowales z ta nowa konfiguracja?
bo to chodzi o to zeby bledy pokazywalo a przy tym -> `"plugins": [["@babel/plugin-proposal-decorators", { "legacy": true }]]` nwm czy dziala